tests/test_instalabel.py

```diff
@@ -0,0 +1,50 @@
+import shippo
+from shippo.models.components import Carriers, DistanceUnitTemplate, WeightUnit, \
+    AddressCreateRequest, ShipmentCreateRequest, InstantTransactionRequestBody, \
+    ParcelCreateRequest
+from tests.helpers_custom import get_carrier_account
+
+
+class TestInstalabel:
+
+    def test_instalabel(self, api: shippo.Shippo):
```

Comment:
Would it make sense to link to the documentation where this use case is described or to present a bit more details in the test name about what is being tested?

Reply:
good call, added doc links